public/js/event-views.js

```diff
@@ -659,7 +1082,7 @@ views.ChatInputView = Backbone.Marionette.ItemView.extend({
 
     chat: function(e) {
         var msg = this.ui.chatInput.val();
-        var postAsAdmin = IS_ADMIN && this.ui.asAdmin.is(":checked");
+        var postAsAdmin = e && this.ui.asAdmin.is(":checked");
```

Comment:
I guess this was not done intentionally. Probably resulted due to mistyping https://github.com/drewww/unhangout/commit/98e6c94058ce8b9a3476f358c462fc98d1425d62